tests/unit/test_email.py

```diff
@@ -1,41 +1,55 @@
 import pytest
 
-from app.jobs.email import send_confirm_email
+from app.jobs.email import send_confirm_email, send_password_change_confirm_email
 from app.services.authentication import encode_token, generate_registration_url
 import requests
 import time
+import json
+import requests
 
 
 class TestEmailJob(object):
     """Email job sends correct email"""
 
     def test_email_job(self, current_user):
         """ The email is sent correctly"""
-        url = "http://mail:8025/api/v2/messages"
-        response = requests.get(url)
-        data = response.json()
-        old_total = data["total"]
+        r = requests.get("http://mail:8025/api/v1/events", stream=True)
 
         token = encode_token(current_user.email)
         confirm_url = generate_registration_url(token)
         send_confirm_email(current_user.email, confirm_url)
 
-        time.sleep(1)
-        response = requests.get(url)
-        data = response.json()
-        new_total = data["total"]
-        assert new_total > old_total
+        if r.encoding is None:
```

Comment:
How does this encoding end up not being set?   If you need it to be `ascii` (plain text mail?) then I think you should just set it to ascii?